Sources/SmithyHTTPAuth/DefaultIdentityResolverConfiguration.swift

```diff
@@ -5,10 +5,10 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
I've made orders of import statements follow lexicographic order, in here and in many following files.

Reply:
IDK, is there a way to have Swiftlint manage this for us?